microsoft-edge/web-platform/how-to-detect-win11.md

```diff
@@ -0,0 +1,102 @@
+---
+# Detecting Windows 11 using User-Agent Client Hints
+
+<!--
+Restrict the lexicon to these forms:
+User-Agent string
+user agent string
+User-Agent Client Hints
+-->
+
+Websites can use user agent information sent from the browser to detect brand, version, device platform, and more. There are two approaches for sites to access user agent information:
+*  User-Agent strings (legacy).
+*  User-Agent Client Hints (recommended).
+
+For details about these two approaches, see [Detecting Microsoft Edge from your website](user-agent-guidance.md).
+
+To differentiate between users who are using Windows 11 and Windows 10, starting in Microsoft Edge version 94 and Chrome version 95, User-Agent Client Hints reflects the following header-field values:
+
+| Header field | Values that indicate Windows 10 | Value that indicates Windows 11 |
+| --- | --- | --- |
+| `Sec-CH-UA-Platform` | `Windows` | `Windows` |
+| `Sec-CH-UA-Platform-Version` | values between `1.0.0` and `10.0.0` | `13.0.0` and above |
+
+User-Agent strings won't be updated to differentiate between Windows 11 and Windows 10.  We don't recommend using User-Agent strings to retrieve user agent data.  Browsers that do not support User-Agent Client Hints will not be able to differentiate between Windows 10 and Windows 11.
+
+
+<!-- ====================================================================== -->
+## Browsers that support User-Agent Client Hints
+
+The following table shows which browsers support differentiating between Windows 11 and Windows 10.
+
+| Browser | Supports differentiation via User-Agent Client Hints? | Alternative detection methods |
+| --- | --- | --- |
+| Microsoft Edge 94+ | Yes | - |
+| Chrome 95+ | Yes | - |
+| Opera | Yes | - |
+| Firefox | No | None |
+| Internet Explorer 11 | No | None |
+| Microsoft Edge Legacy | No | window.external.getHostEnvironmentValue |
+| Safari | No | Not Applicable |
+
+
+<!-- ====================================================================== -->
+## Sample code for detecting Windows 11
+
+The following code detects Windows 11:
+
+```javascript
+navigator.userAgentData.getHighEntropyValues(
+ ["platform",
+  "platformVersion"])
+ .then(ua => {
+   if (ua.platform === "Windows") {
+     const majorPlatformVersion = ua.platformVersion.split('.')[0];
+     if (majorPlatformVersion >= 13) {
+       console.log("Windows 11+");
+      }
+      else if (majorPlatformVersion > 0) {
+        console.log("Windows 10");
+      }
+      else {
+        console.log("Pre-Windows 10");
+      }
+   }
+   else {
+     console.log("Not running on Windows");
+   }
+ });
+
+```
+
+
+<!-- ====================================================================== -->
+## Detecting previous Windows versions
+
+To detect older versions of Windows, use the following values for `platformVersion` in User-Agent Client Hints:
```

Comment:
This is a table of a bunch of versions of Windows, including Window 11. Maybe something like this makes sense:
```suggestion
To detect specific versions of Windows, use the following values for `platformVersion` in User-Agent Client Hints:
```